app/components/provider_interface/offered_course_component.html.erb

```diff
@@ -0,0 +1,6 @@
+<% if display == :course %>
```

Comment:
Having the logic in one component is great, I think we could go further and iron out the `display` param. It feels natural to have `OfferedCourseComponent` compose the rows for `Course` and `Location` rather than depend on callers to tell it which row to show?